docs/source/models/causes/neonatal/index.rst

```diff
@@ -126,30 +133,39 @@ unit time.
 .. graphviz::
 
     digraph NN_decisions {
+        graph [size="7,5"]
```

Comment:
This change seems to have [made the problem worse](https://vivarium-research--1573.org.readthedocs.build/en/1573/models/causes/neonatal/index.html#cause-model-decision-graph) (i.e., now you can barely see any of the graph).

Also... again, why are these changes showing up in this PR when [they have already been merged](https://vivarium-research.readthedocs.io/en/latest/models/causes/neonatal/index.html#cause-model-decision-graph)?

Reply:
I will revert